stylua-vscode/package.json

```diff
@@ -42,6 +42,19 @@
     "configuration": {
       "title": "StyLua",
       "properties": {
+        "stylua.releaseVersion": {
+          "type": "string",
+          "default": "latest",
+          "enum": [
+            "latest",
+            "v0.9",
+            "v0.8",
+            "v0.7",
+            "v0.6",
+            "v0.5"
```

Comment:
We could include older versions in the future by adding support for stylua versions that don't support ranges.

Reply:
Would it be better for this to be a text box rather than a drop down list? There are lots of versions, and more will be added in the future, so the pro here is that it won't get really long, and we don't need to keep remembering to update the extension as more versions are released.
The con is that it means the user needs to input the correct version specifically, which they will need to remember/find from GitHub.

Reply:
This is what I was referencing in my final comment that I'll be adding today. I figured for the average user the dropdown is easier to select a correct version with, but a string to select a specific release tag as an override is good for users with a specific version need including regressions, breaking bugs, or pre-releases.

This also lets us control when older version are no longer being supported officially by the extension (such as versions prior to v0.5 currently). Overall I think this is a good tradeoff. Let me know your thoughts.